src/components/verification-grid/verification-grid.ts

```diff
@@ -305,31 +330,60 @@ export class VerificationGridComponent extends WithShoelace(AbstractComponent(Li
   private keydownHandler = this.handleKeyDown.bind(this);
   private keyupHandler = this.handleKeyUp.bind(this);
   private blurHandler = this.handleWindowBlur.bind(this);
-  private selectionHandler = this.handleSelection.bind(this);
+  private selectionHandler = this.handleTileSelection.bind(this);
   private decisionHandler = this.handleDecision.bind(this);
 
   public subjects: SubjectWrapper[] = [];
 
   private _loaded = false;
   private decisionHeadIndex = 0;
   private viewHeadIndex = 0;
+  private isCurrentAutoSelected = false;
```

Comment:
needs some documentation or a different name.

Is this the field that tracks if we're in decide one at a time mode? I think it is.